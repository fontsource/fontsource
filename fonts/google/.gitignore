# No binary font files
*.ttf
*.otf
*.woff
*.woff2
