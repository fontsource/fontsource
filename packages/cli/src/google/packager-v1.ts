/* eslint-disable no-await-in-loop */
import fs from 'fs-extra';
import { APIv1 } from 'google-font-metadata';
import * as path from 'pathe';

import type { BuildOptions } from '../types';
import { makeFontFilePath } from '../utils';
import { generateV1CSS } from './css';

const packagerV1 = async (id: string, opts: BuildOptions) => {
	const metadata = APIv1[id];
	const cssGenerate = generateV1CSS(metadata, makeFontFilePath);

	for (const item of cssGenerate) {
		const cssPath = path.join(opts.dir, item.filename);
		await fs.writeFile(cssPath, item.css);
	}
};

export { packagerV1 };
