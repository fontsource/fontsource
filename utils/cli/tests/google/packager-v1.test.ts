import fs from 'fs-extra';
import * as gfm from 'google-font-metadata';
import { describe, expect, it, vi } from 'vitest';

import { packagerV1 } from '../../src/google/packager-v1';
import APIv1Mock from './fixtures/google-fonts-v1.json';

vi.mock('fs-extra');
vi.mock('google-font-metadata');

describe('packager v1', () => {
	vi.spyOn(gfm, 'APIv1', 'get').mockReturnValue(APIv1Mock);

	it('should generate Abel CSS successfully', async () => {
		const buildOpts = {
			dir: 'test/abel',
			tmpDir: 'temptest',
			force: false,
			isVariable: false,
		};

		await packagerV1('abel', buildOpts);
		expect(vi.mocked(fs.writeFile)).toMatchSnapshot();
	});

	it('should generate Noto Sans JP successfully', async () => {
		const buildOpts = {
			dir: 'test/noto-sans-jp',
			tmpDir: 'temptest',
			force: false,
			isVariable: false,
		};

		await packagerV1('noto-sans-jp', buildOpts);
		expect(vi.mocked(fs.writeFile)).toMatchSnapshot();
	});

	it('should generate Cabin CSS successfully', async () => {
		const buildOpts = {
			dir: 'test/cabin',
			tmpDir: 'temptest',
			force: false,
			isVariable: true,
		};

		await packagerV1('cabin', buildOpts);
		expect(vi.mocked(fs.writeFile)).toMatchSnapshot();
	});
});
