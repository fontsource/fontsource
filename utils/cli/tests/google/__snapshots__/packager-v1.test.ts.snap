// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`packager v1 > should generate Abel CSS successfully 1`] = `
[MockFunction writeFile] {
  "calls": [
    [
      "test/abel/latin-400.css",
      "/* abel-latin-400-normal */
@font-face {
  font-family: 'Abel';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/abel-latin-400-normal.woff2) format('woff2'), url(./files/abel-latin-400-normal.woff) format('woff');
}",
    ],
    [
      "test/abel/latin.css",
      "/* abel-latin-400-normal */
@font-face {
  font-family: 'Abel';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/abel-latin-400-normal.woff2) format('woff2'), url(./files/abel-latin-400-normal.woff) format('woff');
}",
    ],
  ],
  "results": [
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;

exports[`packager v1 > should generate Cabin CSS successfully 1`] = `
[MockFunction writeFile] {
  "calls": [
    [
      "test/cabin/latin-400-italic.css",
      "/* cabin-latin-400-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/cabin-latin-400-italic.woff2) format('woff2'), url(./files/cabin-latin-400-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-400.css",
      "/* cabin-latin-400-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/cabin-latin-400-normal.woff2) format('woff2'), url(./files/cabin-latin-400-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-500-italic.css",
      "/* cabin-latin-500-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/cabin-latin-500-italic.woff2) format('woff2'), url(./files/cabin-latin-500-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-500.css",
      "/* cabin-latin-500-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/cabin-latin-500-normal.woff2) format('woff2'), url(./files/cabin-latin-500-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-600-italic.css",
      "/* cabin-latin-600-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 600;
  src: url(./files/cabin-latin-600-italic.woff2) format('woff2'), url(./files/cabin-latin-600-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-600.css",
      "/* cabin-latin-600-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 600;
  src: url(./files/cabin-latin-600-normal.woff2) format('woff2'), url(./files/cabin-latin-600-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-700-italic.css",
      "/* cabin-latin-700-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/cabin-latin-700-italic.woff2) format('woff2'), url(./files/cabin-latin-700-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-700.css",
      "/* cabin-latin-700-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/cabin-latin-700-normal.woff2) format('woff2'), url(./files/cabin-latin-700-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin.css",
      "/* cabin-latin-400-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/cabin-latin-400-normal.woff2) format('woff2'), url(./files/cabin-latin-400-normal.woff) format('woff');
}

/* cabin-latin-500-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/cabin-latin-500-normal.woff2) format('woff2'), url(./files/cabin-latin-500-normal.woff) format('woff');
}

/* cabin-latin-600-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 600;
  src: url(./files/cabin-latin-600-normal.woff2) format('woff2'), url(./files/cabin-latin-600-normal.woff) format('woff');
}

/* cabin-latin-700-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/cabin-latin-700-normal.woff2) format('woff2'), url(./files/cabin-latin-700-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-italic.css",
      "/* cabin-latin-400-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/cabin-latin-400-italic.woff2) format('woff2'), url(./files/cabin-latin-400-italic.woff) format('woff');
}

/* cabin-latin-500-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/cabin-latin-500-italic.woff2) format('woff2'), url(./files/cabin-latin-500-italic.woff) format('woff');
}

/* cabin-latin-600-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 600;
  src: url(./files/cabin-latin-600-italic.woff2) format('woff2'), url(./files/cabin-latin-600-italic.woff) format('woff');
}

/* cabin-latin-700-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/cabin-latin-700-italic.woff2) format('woff2'), url(./files/cabin-latin-700-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-ext-400-italic.css",
      "/* cabin-latin-ext-400-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/cabin-latin-ext-400-italic.woff2) format('woff2'), url(./files/cabin-latin-ext-400-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-ext-400.css",
      "/* cabin-latin-ext-400-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/cabin-latin-ext-400-normal.woff2) format('woff2'), url(./files/cabin-latin-ext-400-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-ext-500-italic.css",
      "/* cabin-latin-ext-500-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/cabin-latin-ext-500-italic.woff2) format('woff2'), url(./files/cabin-latin-ext-500-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-ext-500.css",
      "/* cabin-latin-ext-500-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/cabin-latin-ext-500-normal.woff2) format('woff2'), url(./files/cabin-latin-ext-500-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-ext-600-italic.css",
      "/* cabin-latin-ext-600-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 600;
  src: url(./files/cabin-latin-ext-600-italic.woff2) format('woff2'), url(./files/cabin-latin-ext-600-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-ext-600.css",
      "/* cabin-latin-ext-600-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 600;
  src: url(./files/cabin-latin-ext-600-normal.woff2) format('woff2'), url(./files/cabin-latin-ext-600-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-ext-700-italic.css",
      "/* cabin-latin-ext-700-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/cabin-latin-ext-700-italic.woff2) format('woff2'), url(./files/cabin-latin-ext-700-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-ext-700.css",
      "/* cabin-latin-ext-700-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/cabin-latin-ext-700-normal.woff2) format('woff2'), url(./files/cabin-latin-ext-700-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-ext.css",
      "/* cabin-latin-ext-400-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/cabin-latin-ext-400-normal.woff2) format('woff2'), url(./files/cabin-latin-ext-400-normal.woff) format('woff');
}

/* cabin-latin-ext-500-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/cabin-latin-ext-500-normal.woff2) format('woff2'), url(./files/cabin-latin-ext-500-normal.woff) format('woff');
}

/* cabin-latin-ext-600-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 600;
  src: url(./files/cabin-latin-ext-600-normal.woff2) format('woff2'), url(./files/cabin-latin-ext-600-normal.woff) format('woff');
}

/* cabin-latin-ext-700-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/cabin-latin-ext-700-normal.woff2) format('woff2'), url(./files/cabin-latin-ext-700-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/latin-ext-italic.css",
      "/* cabin-latin-ext-400-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/cabin-latin-ext-400-italic.woff2) format('woff2'), url(./files/cabin-latin-ext-400-italic.woff) format('woff');
}

/* cabin-latin-ext-500-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/cabin-latin-ext-500-italic.woff2) format('woff2'), url(./files/cabin-latin-ext-500-italic.woff) format('woff');
}

/* cabin-latin-ext-600-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 600;
  src: url(./files/cabin-latin-ext-600-italic.woff2) format('woff2'), url(./files/cabin-latin-ext-600-italic.woff) format('woff');
}

/* cabin-latin-ext-700-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/cabin-latin-ext-700-italic.woff2) format('woff2'), url(./files/cabin-latin-ext-700-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/vietnamese-400-italic.css",
      "/* cabin-vietnamese-400-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/cabin-vietnamese-400-italic.woff2) format('woff2'), url(./files/cabin-vietnamese-400-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/vietnamese-400.css",
      "/* cabin-vietnamese-400-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/cabin-vietnamese-400-normal.woff2) format('woff2'), url(./files/cabin-vietnamese-400-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/vietnamese-500-italic.css",
      "/* cabin-vietnamese-500-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/cabin-vietnamese-500-italic.woff2) format('woff2'), url(./files/cabin-vietnamese-500-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/vietnamese-500.css",
      "/* cabin-vietnamese-500-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/cabin-vietnamese-500-normal.woff2) format('woff2'), url(./files/cabin-vietnamese-500-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/vietnamese-600-italic.css",
      "/* cabin-vietnamese-600-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 600;
  src: url(./files/cabin-vietnamese-600-italic.woff2) format('woff2'), url(./files/cabin-vietnamese-600-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/vietnamese-600.css",
      "/* cabin-vietnamese-600-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 600;
  src: url(./files/cabin-vietnamese-600-normal.woff2) format('woff2'), url(./files/cabin-vietnamese-600-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/vietnamese-700-italic.css",
      "/* cabin-vietnamese-700-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/cabin-vietnamese-700-italic.woff2) format('woff2'), url(./files/cabin-vietnamese-700-italic.woff) format('woff');
}",
    ],
    [
      "test/cabin/vietnamese-700.css",
      "/* cabin-vietnamese-700-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/cabin-vietnamese-700-normal.woff2) format('woff2'), url(./files/cabin-vietnamese-700-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/vietnamese.css",
      "/* cabin-vietnamese-400-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/cabin-vietnamese-400-normal.woff2) format('woff2'), url(./files/cabin-vietnamese-400-normal.woff) format('woff');
}

/* cabin-vietnamese-500-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/cabin-vietnamese-500-normal.woff2) format('woff2'), url(./files/cabin-vietnamese-500-normal.woff) format('woff');
}

/* cabin-vietnamese-600-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 600;
  src: url(./files/cabin-vietnamese-600-normal.woff2) format('woff2'), url(./files/cabin-vietnamese-600-normal.woff) format('woff');
}

/* cabin-vietnamese-700-normal */
@font-face {
  font-family: 'Cabin';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/cabin-vietnamese-700-normal.woff2) format('woff2'), url(./files/cabin-vietnamese-700-normal.woff) format('woff');
}",
    ],
    [
      "test/cabin/vietnamese-italic.css",
      "/* cabin-vietnamese-400-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/cabin-vietnamese-400-italic.woff2) format('woff2'), url(./files/cabin-vietnamese-400-italic.woff) format('woff');
}

/* cabin-vietnamese-500-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/cabin-vietnamese-500-italic.woff2) format('woff2'), url(./files/cabin-vietnamese-500-italic.woff) format('woff');
}

/* cabin-vietnamese-600-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 600;
  src: url(./files/cabin-vietnamese-600-italic.woff2) format('woff2'), url(./files/cabin-vietnamese-600-italic.woff) format('woff');
}

/* cabin-vietnamese-700-italic */
@font-face {
  font-family: 'Cabin';
  font-style: italic;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/cabin-vietnamese-700-italic.woff2) format('woff2'), url(./files/cabin-vietnamese-700-italic.woff) format('woff');
}",
    ],
  ],
  "results": [
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;

exports[`packager v1 > should generate Noto Sans JP successfully 1`] = `
[MockFunction writeFile] {
  "calls": [
    [
      "test/noto-sans-jp/japanese-100.css",
      "/* noto-sans-jp-japanese-100-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 100;
  src: url(./files/noto-sans-jp-japanese-100-normal.woff2) format('woff2'), url(./files/noto-sans-jp-japanese-100-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/japanese-300.css",
      "/* noto-sans-jp-japanese-300-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 300;
  src: url(./files/noto-sans-jp-japanese-300-normal.woff2) format('woff2'), url(./files/noto-sans-jp-japanese-300-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/japanese-400.css",
      "/* noto-sans-jp-japanese-400-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/noto-sans-jp-japanese-400-normal.woff2) format('woff2'), url(./files/noto-sans-jp-japanese-400-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/japanese-500.css",
      "/* noto-sans-jp-japanese-500-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/noto-sans-jp-japanese-500-normal.woff2) format('woff2'), url(./files/noto-sans-jp-japanese-500-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/japanese-700.css",
      "/* noto-sans-jp-japanese-700-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/noto-sans-jp-japanese-700-normal.woff2) format('woff2'), url(./files/noto-sans-jp-japanese-700-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/japanese-900.css",
      "/* noto-sans-jp-japanese-900-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 900;
  src: url(./files/noto-sans-jp-japanese-900-normal.woff2) format('woff2'), url(./files/noto-sans-jp-japanese-900-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/japanese.css",
      "/* noto-sans-jp-japanese-100-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 100;
  src: url(./files/noto-sans-jp-japanese-100-normal.woff2) format('woff2'), url(./files/noto-sans-jp-japanese-100-normal.woff) format('woff');
}

/* noto-sans-jp-japanese-300-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 300;
  src: url(./files/noto-sans-jp-japanese-300-normal.woff2) format('woff2'), url(./files/noto-sans-jp-japanese-300-normal.woff) format('woff');
}

/* noto-sans-jp-japanese-400-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/noto-sans-jp-japanese-400-normal.woff2) format('woff2'), url(./files/noto-sans-jp-japanese-400-normal.woff) format('woff');
}

/* noto-sans-jp-japanese-500-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/noto-sans-jp-japanese-500-normal.woff2) format('woff2'), url(./files/noto-sans-jp-japanese-500-normal.woff) format('woff');
}

/* noto-sans-jp-japanese-700-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/noto-sans-jp-japanese-700-normal.woff2) format('woff2'), url(./files/noto-sans-jp-japanese-700-normal.woff) format('woff');
}

/* noto-sans-jp-japanese-900-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 900;
  src: url(./files/noto-sans-jp-japanese-900-normal.woff2) format('woff2'), url(./files/noto-sans-jp-japanese-900-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/latin-100.css",
      "/* noto-sans-jp-latin-100-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 100;
  src: url(./files/noto-sans-jp-latin-100-normal.woff2) format('woff2'), url(./files/noto-sans-jp-latin-100-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/latin-300.css",
      "/* noto-sans-jp-latin-300-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 300;
  src: url(./files/noto-sans-jp-latin-300-normal.woff2) format('woff2'), url(./files/noto-sans-jp-latin-300-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/latin-400.css",
      "/* noto-sans-jp-latin-400-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/noto-sans-jp-latin-400-normal.woff2) format('woff2'), url(./files/noto-sans-jp-latin-400-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/latin-500.css",
      "/* noto-sans-jp-latin-500-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/noto-sans-jp-latin-500-normal.woff2) format('woff2'), url(./files/noto-sans-jp-latin-500-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/latin-700.css",
      "/* noto-sans-jp-latin-700-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/noto-sans-jp-latin-700-normal.woff2) format('woff2'), url(./files/noto-sans-jp-latin-700-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/latin-900.css",
      "/* noto-sans-jp-latin-900-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 900;
  src: url(./files/noto-sans-jp-latin-900-normal.woff2) format('woff2'), url(./files/noto-sans-jp-latin-900-normal.woff) format('woff');
}",
    ],
    [
      "test/noto-sans-jp/latin.css",
      "/* noto-sans-jp-latin-100-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 100;
  src: url(./files/noto-sans-jp-latin-100-normal.woff2) format('woff2'), url(./files/noto-sans-jp-latin-100-normal.woff) format('woff');
}

/* noto-sans-jp-latin-300-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 300;
  src: url(./files/noto-sans-jp-latin-300-normal.woff2) format('woff2'), url(./files/noto-sans-jp-latin-300-normal.woff) format('woff');
}

/* noto-sans-jp-latin-400-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 400;
  src: url(./files/noto-sans-jp-latin-400-normal.woff2) format('woff2'), url(./files/noto-sans-jp-latin-400-normal.woff) format('woff');
}

/* noto-sans-jp-latin-500-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 500;
  src: url(./files/noto-sans-jp-latin-500-normal.woff2) format('woff2'), url(./files/noto-sans-jp-latin-500-normal.woff) format('woff');
}

/* noto-sans-jp-latin-700-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 700;
  src: url(./files/noto-sans-jp-latin-700-normal.woff2) format('woff2'), url(./files/noto-sans-jp-latin-700-normal.woff) format('woff');
}

/* noto-sans-jp-latin-900-normal */
@font-face {
  font-family: 'Noto Sans JP';
  font-style: normal;
  font-display: var(--fontsource-display, swap);
  font-weight: 900;
  src: url(./files/noto-sans-jp-latin-900-normal.woff2) format('woff2'), url(./files/noto-sans-jp-latin-900-normal.woff) format('woff');
}",
    ],
  ],
  "results": [
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
    {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;
